var/spack/repos/builtin/packages/hermes/package.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Hermes(CMakePackage):
+    """
+    Hermes is a heterogeneous-aware, multi-tiered, dynamic, and distributed
+    I/O buffering system that aims to significantly accelerate I/O performance.
+    """
+
+    homepage = "http://www.cs.iit.edu/~scs/assets/projects/Hermes/Hermes.html"
+    git = "https://github.com/HDFGroup/hermes.git"
+
+    maintainers = ['hyoklee']
+
+    version('master', branch='master')
+    version('0.4.0-beta', url="https://github.com/HDFGroup/hermes/archive/v0.4.0-be\
+ta.tar.gz", sha256='e8bd0b701149844f7f113a55a107e40b3579945d70ca2cb8ce9db5b5a131efd\
+b')
+
+    variant('vfd', default=False, description='Enable HDF5 VFD')
+
+    depends_on('mochi-thallium~cereal@0.8.3')
+    depends_on('catch2@2.13.3')
```

Comment:
These are very specific versions. Are you sure only these exact ones will work?

Reply:
Yes, for now. We will relax as we test more with the latest versions.

Reply:
I strongly suggest writing these as 
```suggestion
    depends_on('mochi-thallium~cereal@0.8:')
    depends_on('catch2@2.13:')
```
unless there are known incompatibilities with the earlier patch versions or expected incompatibilities with future versions of the software.